### PR TITLE
Bump atlas-bash-util to v0.2.2

### PR DESCRIPTION
**Pull request by pmb59**

### CODE REVIEW

File: recipes/atlas-bash-util/build.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
 mkdir -p $PREFIX/bin
-cp *.sh atlas-bash-util atlas-lsf $PREFIX/bin
+cp *.sh atlas-bash-util atlas-lsf atlas-slurm $PREFIX/bin
```
Comment (anilthanki) [inline recipes/atlas-bash-util/build.sh:2]:
```suggestion
cp *.sh atlas-bash-util atlas-lsf $PREFIX/bin
```

Comment (anilthanki) [reply recipes/atlas-bash-util/build.sh:2]: We dont need to copy `atlas-slurm` anymore

Comment (pmb59) [reply recipes/atlas-bash-util/build.sh:2]: I think we have to do it, as it was done before for `atlas-lsf`

Comment (anilthanki) [reply recipes/atlas-bash-util/build.sh:2]: ohh my bad.. I am wrong

**Review by anilthanki — APPROVED** [review]

**Pull request merged**